.ci/docker/common/install_conda.sh

```diff
@@ -85,6 +83,11 @@ if [ -n "$ANACONDA_PYTHON_VERSION" ]; then
   # Install some other packages, including those needed for Python test reporting
   pip_install -r /opt/conda/requirements-ci.txt
 
+  # Install PyTorch mkl deps, as per https://github.com/pytorch/pytorch README
+  if [[ $(uname -m) != "aarch64" ]]; then
+    pip_install mkl-static==2024.2.0 mkl-include==2024.2.0
```

Comment:
This logic is duplicated: https://github.com/pytorch/pytorch/blob/c2e91157575a6064e19f2701e85af4035d33a23a/.ci/docker/common/install_mkl.sh#L5

Also, does mkl-static reallly distribute static libs? Won't that cause toolchain issues, especially if we want to enable Link Time Optimization on PyTorch at some point?

Reply:
Thanks for your comments. I don't know the relationship between `pytorch/.ci/docker/common/install_mkl.sh` and the MKL version of the CI environment. Does `pytorch/.ci/docker/common/install_mkl.sh` selects the MKL version for the pytorch releases ? I'm not sure whether such changes will cause the issues you mentioned. I tried to update MKL according to https://github.com/pytorch/pytorch README. Could you please give me some advice? Thanks.